components/os/component_output.go

```diff
@@ -150,6 +166,15 @@ func ParseStatesToOutput(states ...components.State) (*Output, error) {
 			}
 			o.Uptimes = uptimes
 
+		case StateNameProcessCountsByStatus:
+			counts, err := ParseStateProcessCountsByStatus(state.ExtraInfo)
```

Comment:
its gonna be messy if we cumulate all possible process states...R,S,D,T,Z with suffix...maybe we should just count Zombie process count in extra info and return unhealthy if exceed threshold..